deployment/README.md

```diff
@@ -63,83 +102,115 @@ The deployer must run under a service account defined as follows:
+If it seems to be taking too long, you can retrieve more details about the pod and
+any associated events with:
 
-Enable fluentd service account - edit SCC with the following
+    oc describe pod/<pod-name>
 
-    oc edit scc/privileged
+When it runs, check the logs of the resulting pod (`oc logs -f <pod name>`)
+for some instructions to follow after deployment. More details
+are given below.
 
-Add one line as the user at the end: (note, change `:logging:` below to the project of your choice)
-
-    - system:serviceaccount:logging:aggregated-logging-fluentd
+## Deploy the templates created by the deployer
 
-Give the account access to read labels from all pods:
+### Supporting definitions
 
-    openshift admin policy add-cluster-role-to-user cluster-reader system:serviceaccount:logging:aggregated-logging-fluentd
+Create the supporting definitions from template (you must be cluster admin):
 
+    oc process logging-support-template | oc create -f -
 
-### ElasticSearch
+Tip: Check the output to make sure that all objects were created
+successfully. If any were not, it is probably because one or more
+already existed from a previous deployment (potentially in a different
+project). You can delete them all before trying again:
 
-Scaling a deployment today assumes that all pods can safely share
-any volumes specified in the deployment. This is not the case with
-ElasticSearch; each pod requires its own storage. Work is under way
-to enable specifying multiple volumes to be allocated to instances in
-a deployment, but for now multiple deployments are used in order to
-scale ElasticSearch. You can view them all with:
+    oc process logging-support-template | oc delete -f -
 
-    oc get dc --selector logging-infra=elasticsearch
+### ElasticSearch
 
-It is possible to scale your cluster up after creation by adding more
-deployments; however, when scaling up (or down), you must be aware of
-cluster parameters that vary by cluster size and adjust them accordingly
-for both new and existing deployments. Elastic [discusses these issues
-here](https://www.elastic.co/guide/en/elasticsearch/guide/current/_important_configuration_changes.html)
-and the corresponding parameters are coded into the deployments and the
-template below.
+The deployer creates the number of ElasticSearch instances specified by
+`ES_CLUSTER_SIZE`. The nature of ElasticSearch and current Kubernetes
+requires that we use a different scaling mechanism than the standard
+Kubernetes scaling.
 
-The deployer defines a template which it uses to create ElasticSearch
-deployments. You can adjust and reuse it to add more:
+Scaling a deployment (a Kubernetes ReplicationController) to
+multiple pods currently has all pods share any volumes specified in
+the deployment. Multiple ElasticSearch instances in a cluster cannot
+do this; each pod requires its own storage. Work is under way to enable
+specifying multiple volumes to be allocated to instances in a deployment,
```

Comment:
"...in a deployment.  For now, the deployer creates multiple deployments in order.."  NOTE: deletion of 'are'